### DISCUSSION TITLE
How to place bets on Cloudbet.com programmatically

### DISCUSSION THREAD

**Discussion by cuzmuc** [cloudbet/docs#90] [Q&A]
Hello, everyone!

I am Cuzmuc Ghennadii from Moldova.
I am going to place bets on some betting sites like Cloudbet.com, bet365.com, matchbook.com and betfair.com programmatically using node.js project.
I tried some time, but the result is not so good.
So, I want to get help from you.
If you have any experience in this field, looking forward to hearing from you.

Best regards, Cuzmuc Ghennadii

**Comment by kgravenreuth** [comment]
Hi @cuzmuc  ,
Thanks for reaching out to us. We have several resources to help you get started with the Cloudbet API. Feel free to refer to them, and we hope you'll soon be placing bets with the Cloudbet API!

- [Cloudbet Feed API](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis) and [Cloudbet Trading API](https://www.cloudbet.com/api/?urls.primaryName=Trading&utm_source=github.com&utm_medium=community&utm_campaign=axis) Documentation
- You can also refer to our [Cloudbet API Tutorial blog post](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang?utm_source=github.com&utm_medium=community&utm_campaign=axis) which guides you through Account creation as well as how to obtain pricing data and start betting on the Cloudbet API.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by cuzmuc** [comment]
Thanks for your kind explanation and offer!!!
Have a nice day!

Cuzmuc G
